### PR TITLE
ddl: refine cancel drop table/schema code.

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
This PR try to keep the code style consistent with other rolling back logic.

### Check List 

Tests 

Code changes

Side effects

Related changes

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8858)

### CODE REVIEW

File: ddl/column.go [hunk 1]
```diff
@@ -148,15 +148,6 @@ func checkAddColumn(t *meta.Meta, job *model.Job) (*model.TableInfo, *model.Colu
...
 
 func onAddColumn(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error) {
-	// Handle the rolling back job.
-	if job.IsRollingback() {
-		ver, err = onDropColumn(t, job)
-		if err != nil {
-			return ver, errors.Trace(err)
-		}
-		return ver, nil
-	}
```
[the remark this answers is not in the document]
Comment (crazycs520) [reply ddl/column.go]: done. PTAL

File: ddl/column.go [hunk 2]
```diff
@@ -121,42 +121,43 @@ func createColumnInfo(tblInfo *model.TableInfo, colInfo *model.ColumnInfo, pos *
 	return colInfo, position, nil
 }
 
-func onAddColumn(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error) {
-	// Handle the rolling back job.
-	if job.IsRollingback() {
-		ver, err = onDropColumn(t, job)
```
Comment (zimulala) [inline ddl/column.go]: Could we call `onDropColumn` in `rollingbackAddColumn` directly? Then we needn't write the logic of `onDropColumn` again.

**Pull request merged**